pkg/server/commands/reverseexpand/reverse_expand_weighted_test.go

```diff
@@ -935,6 +935,49 @@ func TestReverseExpandWithWeightedGraph(t *testing.T) {
 			expectedOptimizedObjects:   []string{"org:a"},
 			expectedUnoptimizedObjects: []string{"org:a", "org:c"},
 		},
+		{
+			name: "ttus_with_multiple_parents",
```

Comment:
Is there no matrix for LO? Ideally this should be covered at a high level if we make yet another strategy/refactor.